qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/commons/QuteJavadocParams.java

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+* Copyright (c) 2022 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.commons;
+
+/**
+ * Represents the parameters needed to fetch the javadocs from the Java language
+ * server component.
+ * 
+ * @author datho7561
+ */
+public class QuteJavadocParams {
```

Comment:
It should be nice to use the same name than QuteDefinitionParams. For instance using sourceType instead of className.

QuteJavadocParams could extends QuteDefinitionParams?

Reply:
I don't think there is enough overlap in the types for one to extend the other, but I'll rename the `className` to `sourceType` to be consistent